src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -50,40 +50,47 @@ def test_dHdl_invalidfiles(invalid_file):
     assert extract_dHdl(invalid_file, T=298.0) is None
 
 
-def test_dHdl_time_reading(single_dHdl, first_time=22.0, last_time=1020.0):
+def test_dHdl_time_reading(single_dHdl):
     """Test if time information is read correctly when extracting dHdl"""
     dHdl = extract_dHdl(single_dHdl, T=298.0)
-    assert_allclose(dHdl.index.values[0][0], first_time)
-    assert_allclose(dHdl.index.values[-1][0], last_time)
+    assert_allclose(dHdl.index.values[0][0], 22.0)
```

Comment:
I like these way of just put the numbers there.

Reply:
Yeah, I used the other way before just because other already existing tests were made like that, but I think it's cleaner like that...